app/UI/message-panel.tsx

```diff
@@ -192,6 +195,15 @@ export function ParseMessageContent(
         getEventByID: func_GetEventByID;
     },
 ) {
+    if (message.is_deleted) {
+        return (
+            <div class={`text-[#A3A6AA]`}>
+                <p class={`italic`}>
+                    This message was deleted
+                </p>
+            </div>
+        );
+    }
```

Comment:
Let's not have it and actually filter out deleted messages when constructing the parent view